frappe/__init__.py

```diff
@@ -1536,7 +1536,7 @@ def make_property_setter(
 			.select(DocField_doctype.parent)
 			.where(DocField_doctype.fieldname == args.fieldname)
 			.distinct()
-		).run(as_list=True)
+		).run(as_list=True, pluck=True)
```

Comment:
```suggestion
		).run(pluck=True)
```

This should do?

Reply:
Yeah. 

Should we merge these two make_propert_setters too? 😅